community/examples/hpc-slurm-h4d.yaml

```diff
@@ -41,18 +41,12 @@ deployment_groups:
       network_routing_mode: REGIONAL
       enable_cloud_router: false
       enable_cloud_nat: false
+      enable_internal_traffic: false
```

Comment:
why is this set to false? Wouldn't this lead to no node intercomunication? or is RDMA profile already enabling node to node traffic?

Reply:
Correct currently these [firewall rules](https://cloud.google.com/vpc/docs/rdma-network-profiles#additional_features_that_dont_apply_to_traffic_from_rdma_nics) shouldn't affect any RDMA traffic.